Csm-scrum-theory-guide.md

```
@@ -0,0 +1,417 @@
+9. [Scrum Theory 全体像のまとめ図](#8-scrum-theory-全体像のまとめ図)
+10. [理解度チェック（練習問題）](#9-理解度チェック練習問題)
+11. [まとめ：学習到達度チェックリスト](#10-まとめ学習到達度チェックリスト)
+12. [参考文献・ソース一覧](#11-参考文献ソース一覧)
+
+---
+
+## 0. このガイドについて
+
+### 0.1 CSM®試験の全体像
+
+CSM®は、Scrum Alliance®が提供する入門レベルのScrumマスター認定資格です。認定講師（CST®）による16時間のトレーニングコースを受講した後、オンラインの選択式試験（50問中37問正解＝74%以上で合格）を受験します。試験は1時間以内に完了する必要があり、コース修了後90日以内に2回まで受験できます（コース費用に2回分の受験権が含まれます）[4][14]。
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "14"

(MD052, reference-links-images)